data/crew.txt

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2019 by Luke Arndt
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+crew "default"
+  name "Regulars"
+  salary 100
+
+# crew "pilot"
+#   name "Pilots"
+#   salary 250
+#   "avoids flagship"
+#   "minimum per ship" 1
+# 
+# crew "junior officer"
+#   name "Junior Officers"
+#   salary 500
+#   "population per member" 5
```

Comment:
If we want to incorporate Zitchas remarks, then we need to differentiate between `ship-population per member` and `fleet-population per member`.

Reply:
Good point. The variable is now called "ship population per member". I like the new name better, it's clearer.